src/Groups/matrices/iso_nf_fq.jl

```diff
@@ -0,0 +1,172 @@
+# Detinko, Flannery, O'Brien "Recognizing finite matrix groups over infinite
+# fields", Section 4.2
```

Comment:
Could you add a reference for this (and anything else where it might make sense) to the .bib file in the `docs` dir?